transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/CctTransportBackend.java

```diff
@@ -243,7 +243,8 @@ private HttpResponse doSend(HttpRequest request) throws IOException {
       }
       channel.write(ByteBuffer.wrap(output.toByteArray()));
       int responseCode = connection.getResponseCode();
-      LOGGER.info("Status Code: " + responseCode);
+      Logging.i(LOG_TAG, "Status Code: " + responseCode);
```

Comment:
Use %s instead of +

Reply:
Yeah, so I decided not to add an overload that takes format args and did not want to use String.format() directly here as this case is too simple. As we add more uses of info logging we can revisit this.